bubuku/features/terminate.py

```diff
@@ -29,6 +29,29 @@ def can_run_at_exit(self):
         return True
 
 
+class CompleteStopChange(Change):
```

Comment:
as I understand the diff between `CompleteStopChange` and `StopBrokerChange` is 

```
    def can_run_at_exit(self):
         return False
```

maybe inheriting StopBrokerChange is better